driver-core/src/main/java/com/datastax/driver/core/policies/LatencyAwarePolicy.java

```diff
@@ -367,20 +371,45 @@ public long getMeasurementsCount() {
         }
     }
 
+    /**
+     * A set of DriverException subclasses that we should prevent from updating the host's score.
+     * The intent behind it is to filter out "fast" errors: when a host replies with such errors,
+     * it usually does so very quickly, because it did not involve any actual
+     * coordination work. Such errors are not good indicators of the host's responsiveness,
+     * and tend to make the host's score look better than it actually is.
+     */
+    private static final Set<Class<? extends DriverException>> EXCLUDED_EXCEPTIONS = ImmutableSet.of(
```

Comment:
Hrmm, it's hard to tell which will fail early.  I'd also expect 'SyntaxError', 'InvalidQueryException', and 'InvalidConfigurationInQueryException' to be raised early as well since the query would fail to prepare on the coordinator very early.